scripts/start/submit.R

```diff
@@ -128,9 +128,13 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
   if (grepl("^direct", cfg$slurmConfig) || ! isSlurmAvailable()) {
     exitCode <- system("Rscript prepareAndRun.R")
   } else {
+    if (file.exists("log.txt")) {
+      message("\nRestarting REMIND, find old log in 'log_beforeRestart.txt'.")
+      file.rename("log.txt", "log_beforeRestart.R")
+    }
```

Comment:
I don't get why this is here: if restarted after preemption this will not be executed anyway (as you correctly corrected me ;-)), and if the run is restarted by the user the log and other files are copied [here](https://github.com/remindmodel/remind/blob/2758f32fe4907ad33f575f45aff497580f949012/start.R#L193), aren't they?

Reply:
Yes, that is true.